qemu_mode/patches/afl-qemu-cpu-inl.h

```diff
@@ -293,7 +293,8 @@ static void print_mappings(void) {
 
 void afl_forkserver(CPUState *cpu) {
 
-  static unsigned char tmp[4];
+  u32                  map_size = 0;
+  static unsigned char tmp[4] = {0, 0, 0, 0};
```

Comment:
This is still static?

Reply:
I didnt want to touch it as this is Andreas domain but yeah ... just removed it